### PR TITLE
[NEW UI] Add loader on welcome page button

### PR DESCRIPTION
**Pull request by tblivet**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Add loader on welcome page button when clicking on "Get started"
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | -
| Sponsor company   | @PrestaShopCorp
| How to test?      | Go to the welcome page in the module (NEW UI), select an option, and click 'Get Started.' A loader should appear on the button until the page changes.

### CODE REVIEW

**Review by Quetzacoalt91 — APPROVED** [review]

**Review by AureRita — APPROVED** [review]
Hi @tblivet 

Thank you for your PR, I tested it and it seems to works as you can see : 

[Uploading recording(326).webm…]()

Tested from : 
1.7.7.8 to 8.2 
8.0.4 to 8.2 
8.2 to 9.0.0
8.0.4 to 9.0.0

we see that if we return in the page we didn"t change to another one as you can see : 


[recording(327).webm](https://github.com/user-attachments/assets/0f5aad61-d848-4cbf-bfec-203646123e00)

but we see together than it not because of your PR so because the PR seems to works as expected, It's QA :heavy_check_mark: 

Thank you

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
